### PR TITLE
Adding more German translations and updating i18n Readme

### PR DESCRIPTION
**Pull request by bitnapper**
Closes #

### CODE REVIEW

File: openlibrary/i18n/de/messages.po [hunk 1]
```diff
@@ -2185,9 +2193,66 @@ msgstr "(Optional, aber sehr nützlich!)"
 #: openlibrary/macros/EditButtons.html:13
 msgid ""
 "By saving a change to this wiki, you agree that your contribution is given freely to the world under <a href=\"https://"
-"creativecommons.org/publicdomain/zero/1.0/\" target=\"_blank\" title=\"This link to Creative Commons will open in a new window"
+"creativecommons.org/publicdomain/zero/1.0/\" target=\"_blank\" title=\"Dieser Link aufCreative Commons wird in einem neuen Fenster geöffnet"
```
Comment (cdrini) [inline openlibrary/i18n/de/messages.po:2196]: This is the English part :) 

```suggestion
"creativecommons.org/publicdomain/zero/1.0/\" target=\"_blank\" title=\"This link to Creative Commons will open in a new window"
```

**Comment by cdrini** [comment]
Hmm, it looks like a number of these aren't showing up on the site :/ I think we should merge this (once that fix is added), but if you have the chance, could you try taking a look at the docs on https://github.com/internetarchive/openlibrary/pull/3725 ? I tried to clean them up / fix them to avoid errors. Would love to see if you can get it working!

**Comment by bitnapper** [comment]
I tried to get `./scripts/i18n-messages extract` running to update the pot an po files but It does not work. I installed the missing modules but still it says it s not available. I tried to install docker as it seems in these containers the env is setup with all deps but at the moment I cant figure it out how to get running at al. so it will take a while until I can continue.

**Comment by bitnapper** [comment]
Your changes on the readme are very helpful. I made the mistake to read the docker Readme first when I should just have done what is in the i18n readme.

But the i18n-messages seems to create po not meeting the specs. This leads to trouble with the editor. In case of gtranslator it denied me saving my changes until the error was fixed, not telling me which line was causing the error. After running into this trap, there is an easy workaround:

- execute `msgfmt -v -c -o /dev/null messages.po`  on the original file
- looking up the affected line and remember the string
- look up the string with the search in gtranslator and
- fix the line
- saving your file

**Comment by bitnapper** [comment]
I tried to revert my Version of the README.md not to substitute your. I'm not sure if I did the opposite. So just reject my Version of this file.

**Comment by cdrini** [comment]
Hi @bitnapper ! Sorry for the delay. This PR has become a little tangled 😓 We can't change the `msgid` strings in the .po files; that's what's used to fetch the translated string. The .po files should only change the `msgstr`. There have been some conflicts as well now. I think there might be a way to detangle this, though. I'll see if I can find a way :+1:

**Comment by mekarpeles** [comment]
I think the right solution is for us to basically clobber all changes except for the `de` ones and play those on top of `master`; I think @jimchamp is on this when there's time.

ty!

**Pull request closed without merging**